docs/release-notes/RELEASE-NOTES-2.0.0-rc.1.md

```diff
@@ -1,70 +1,105 @@
 # About this release
 
-[This][this-release] is an experimental Minerva release.
+[This][this-release] is the first Minerva release candidate.
 It:
+* Does all the things mentioned temporarily in files [/docs/release-notes/next/PT-*.md](/docs/release-notes/next/).
 * Does all the things mentioned temporarily in files [/docs/release-notes/next-minerva/PT-*.md](/docs/release-notes/next-minerva/).
 
+TODO: When preparing the release, concatenate all `/docs/release-notes/next/*` Markdown files and place them in this file. (Hint: you can use auxiliary script `scripts/cat-files-in-directory-sorted-by-committer-date` and command `git log -p -w --color-moved`.)
 TODO: When preparing the release, concatenate all `/docs/release-notes/next-minerva/*` Markdown files and place them in this file. (Hint: you can use auxiliary script `scripts/cat-files-in-directory-sorted-by-committer-date` and command `git log -p -w --color-moved`.)
 
 [this-release]: https://github.com/aeternity/epoch/releases/tag/v2.0.0-rc.1
 
 This release is not backward compatible with v1.*.
 
+Please join the testnet by following the instructions below, and let us know if you have any problems by [opening a ticket](https://github.com/aeternity/aeternity/issues).
```

Comment:
@sennui @dincho @gorillainduction @evdoks  Please notice invitation to join testnet because RC1. Open to suggestions.

Reply:
LGTM